File: app/src/main/java/org/tvheadend/tvhclient/ui/features/dvr/RecordingConfigSelectedListener.java
package org.tvheadend.tvhclient.ui.features.dvr;

import org.tvheadend.tvhclient.domain.entity.Channel;

public interface RecordingConfigSelectedListener {

    void onChannelSelected(Channel channel);
    
    void onProfileSelected(int which);

    void onPrioritySelected(int which);

    void onDaysSelected(int selectedDays);
}

File: app/src/main/java/org/tvheadend/tvhclient/ui/features/dvr/RecordingConfigSelectedListener.kt
package org.tvheadend.tvhclient.ui.features.dvr

import org.tvheadend.data.entity.Channel

interface RecordingConfigSelectedListener {

    fun onChannelSelected(channel: Channel)

    fun onProfileSelected(which: Int)

    fun onPrioritySelected(which: Int)

    fun onDaysSelected(selectedDays: Int)
}
